### PR TITLE
Added local backup capabilities

### PR DESCRIPTION
**Pull request by Michaelcraun**
- Removed path arg for backup command
- Finished logic for scheduling backup
- Implemented Backup class
    - Implemented local backup
    - Implemented local clean

### CODE REVIEW

**Comment by Michaelcraun** [comment]
Please note that I did NOT hook this up to a cron job as that functionality has not yet been fleshed out.

File: util/configuration.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from .mielib.custominput import choice_input, regex_input, range_input, \
     int_input
-from .cron import CronDate, CronFrequency
```
Comment (BAChapin) [inline util/configuration.py:3]: Why did you remove these imports?

Comment (BAChapin) [reply util/configuration.py:3]: Are conflicts with another import? Because it might cause an issue if I can't use those moving forward.

File: main.py [hunk 2]
```diff
@@ -192,7 +186,7 @@ def main():
         "latest version.", dest="update", nargs="?", const="", required=False)
 
     parser.add_argument('-bu', '--backup', help="Backup your Minecraft Server", 
-        dest="path", nargs="?", const="Default", type=str, required=False)
+        dest="path", action='store_true', required=False)
```
Comment (BAChapin) [inline main.py:189]: So you think giving the user, which I realize is use at present, the ability to choose a location to do a manual backup offers no value? Or do you think it would be better down the road?

File: util/configuration.py (hunk 1 above)
Comment (Michaelcraun) [reply util/configuration.py:3]: Those should not have been removed... They should have been commented. They were causing an issue with my setup because they are not finished and unused in that file.

File: main.py (hunk 2 above)
Comment (Michaelcraun) [reply main.py:189]: Given that they can change where the backups are stored via the config.yml, I don't really see a benefit in it, no.

**Comment by Michaelcraun** [comment]
Fyi, I also couldn't get it to work using `~/MC_Backups`. It kept throwing a FileNotFound error.

**Review by BAChapin — APPROVED** [review]
